decidim-accountability/app/views/decidim/accountability/results/preview.html.erb

```diff
@@ -0,0 +1,27 @@
+<% content_for :js_content do %>
+  <%= javascript_pack_tag "decidim_accountability" %>
+<% end %>
+
+<% content_for :css_content do %>
+  <%= stylesheet_pack_tag "decidim_accountability", media: "all" %>
+<% end %>
+
+<%= turbo_frame_tag "main_frame" do %>
+  <% content_for :aside do %>
+    <%= render partial: "projects_aside" %>
+  <% end %>
+
+  <%= render layout: "layouts/decidim/shared/layout_two_col" do %>
+    <% if category.present? %>
+      <h2 class="h4"><%= translated_attribute category.name %></h2>
+    <% else %>
+      <h2 class="h4"><%= t("decidim.accountability.results.home_header.global_status") %></h2>
+    <% end %>
+
+    <div id="results" class="my-20">
+      <%= t("decidim.accountability.results.preview.loading") %>
+    </div>
+  <% end %>
+<% end %>
+
+<%# <%= turbo_frame_tag "project_frame" %1> %>
```

Comment:
@entantoencuanto can this comment be removed?

Reply:
Yes, I think this frame is not necessary in this view. I'm going to remove it also in meetings